addons/account/models/account_move_line.py

```diff
@@ -3247,6 +3247,7 @@ def _update_analytic_distribution(self):
         for line in self:
             line.with_context(skip_analytic_sync=True).analytic_distribution = {
                 analytic_line._get_distribution_key(): -analytic_line.amount / line.balance * 100
+                if line.balance else 0
```

Comment:
The distribution is certainly not 0
```suggestion
                if line.balance else 100
```

Reply:
Hello @william-andre :wave: 
Is it okay now ?